### PR TITLE
[BOUNTY] New help emotes

### LINKED ISSUE #838: New Help intent emotes (Core)
Requesting the addition of targetted emotes while on help intent with an empty hand, instead of always hugging. Below are examples, and not specifically what I'm requesting - I'd just really like to see this implemented in some fashion.

Targeting Head/Eyes = $NAME pats you on the head
Targeting Arm = $NAME shakes your hand or $NAME high fives you
Targeting Torso = $Name hugs you to make you feel better
Targetting Groin = (To the player attempting only) ERP is not allowed on this server
Targetting Leg = ??? Can't think of anything in particular

Idea is not my own, entirely fueled by this thread:
https://forums.beestation13.com/t/new-help-intent-emotes/989/16

### PR DESCRIPTION
**Pull request by ike709**
Closes #838 

## Changelog
:cl:
add: New help emotes depending on the zone selected.
/:cl:

### CODE REVIEW

**Comment by ike709** [comment]
Still need to test it.

**Comment by ike709** [comment]
Also forgot to do the groin one.

File: code/modules/mob/living/carbon/carbon_defense.dm [hunk 1]
```diff
@@ -293,6 +293,15 @@
...
 			trauma.on_hug(M, src)
+	else if(M.zone_selected == BODY_ZONE_HEAD)
+		M.visible_message("<span class='notice'>[M] pats [src] on the head.</span>", \
+					"<span class='notice'>You pat [src] on the head.</span>")
+	else if((M.zone_selected == BODY_ZONE_L_ARM) || (M.zone_selected == BODY_ZONE_R_ARM))
+		if(!get_bodypart(check_zone(M.zone_selected)))
+			to_chat(M, "<span class='warning'>[src] does not have a [M.zone_selected == BODY_ZONE_L_ARM ? "left" : "right"] arm!</span>")
+		else
+			M.visible_message("<span class='notice'>[M] shakes [src]'s hand'.</span>", \
+						"<span class='notice'>You pat [src] on the head.</span>")
```
Comment (Cenrus) [inline code/modules/mob/living/carbon/carbon_defense.dm]: That should read "You shake [src] hand"

**Comment by ike709** [comment]
Works fine locally.

**Comment by ike709** [comment]
@Rukofamicom

**Comment by Cenrus** [comment]
I feel that the groin targeting message is unnecessary, it feels too OOC in IC, but it is Rukofamicon's choice.

**Comment by qwertyquerty** [comment]
I sign off on the groin one, it funne

**Comment by ike709** [comment]
I feel gross self-merging but Qwerty's fine with it and the bounty creator DMed me thanks.

**Pull request merged**